bittide-instances/src/Bittide/Instances/Tests/FullMeshHwCc.hs

```diff
@@ -260,13 +342,8 @@ fullMeshHwCcTest refClkDiff sysClkDiff syncIn rxns rxps miso =
       False
       sysClk
 
-      -- Consider test done if clock control has been stable consistently for 5 s
-      (trueFor5s sysClk testRst allStable)
-
-      -- This test either succeeds or times out, so success is set to a static
-      -- 'True'. If you want to see statistics, consider setting it to 'False' -
-      -- it will make the test TCL print out all probe values.
-      (pure True :: Signal Basic125 Bool)
+      (trueFor5s sysClk testRst allStable .||. transceiversFailedAfterUp) -- done
```

Comment:
Let's also not put logic in the arguments of large VIOs, but rather give proper names and put the logic there.